jwst/ramp_fitting/ramp_fit_step.py

```diff
@@ -1,8 +1,15 @@
 #! /usr/bin/env python
 
+import numpy as np
```

Comment:
Here process( ) is calling stcal's ramp_fit( ) with a  JWST RampModel as an input argument - I thought the plan was to call the common ramp_fit() with either a generic datamodel or numpy arrays; perhaps the JWST RampModel is generic enough.

Also, a minor comment: we should update -    
 # Only algorithm allowed for Build 7.1
  # Only weighting allowed for Build 7.1

Reply:
For the moment, the assumption is that the RampModel will be common, or at least common enough with relevant ramp fitting key words being the same, across pipelines.  I agree that the documentation should clearly state the only algorithms available right now are ordinary least squares with "optimal" weighting.